game.js

```diff
@@ -94,6 +95,29 @@ $(function () {
         }
     };
 
+    const potion = {
+        energy: 200,
+        thisImage: 0,
+        images: [
+            `${imgPathPotion}potion.png`,
+        ],
+        location: {
+            x: 600,
+            y: -100,
+        },
+        walk: function (moveX, moveY) {
+            this.location.x += moveX;
+            this.location.y += moveY;
+        },
+        strike: function (deplete) {
+            console.log(this.energy);
+            this.energy -= deplete;
+        },
+        pickUpEquipment: function (item) {
+            this.equipment.push(item);
+        }
+    };
+
```

Comment:
The code for Dragon, Potion and Warrior looks similar enough that it warrants a refactoring. Consider creating a base character prototype and extending it to avoid code repetition.

Reply:
I will make this refactoring in the next pull request. Could we close the issues on this one?

Reply:
Ok, that makes sense.